build_web_compilers/lib/src/loader.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// For apps compiled both with dart2js and dart2wasm, this entrypoint script
+// is responsible for detecting browser features and then loading either the
+// wasm or the JS bundle depending on which features are supported.
+//
+// Note: build_web_compiler loads a minified version of this file. To re-generate
+// it, run:
+//
+// esbuild lib/src/loader.js --bundle "--external:*.mjs" --format=esm --minify > lib/src/loader.min.js
```

Comment:
I don't know what `esbuild` is or how to set it up :). Ideally I would like to avoid adding dependencies like this into the build process.  This file is only 40 lines so it is probably fine to just include it unminified?

Reply:
agree, probably not worth minifying at this time (esbuild is popular and fast JS-to-JS compiler/minifier, see https://esbuild.github.io/)